experience.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <link rel="stylesheet" type="text/css" href="css/styles.css">
+        <title>Beyonce 101: Experience</title>
+        <meta charset="utf-8">
+        <meta name="description" content="This lists the accomplishments of Beyonce for this front-end web development class">
+    </head>
+
+      <body>  
+          <nav>
+            <li><a href="index.html">Home</a></li>
+            <li class="second-nav"><a href="experience.html">Experience</a></li>
+            </div>
```

Comment:
@kvelez1  please remove `</div>`, the div is not used anywhere 🔨

Reply:
@rmjames removed </div>!